modules/claims_api/lib/bd/bd.rb

```diff
@@ -123,16 +122,22 @@ def generate_upload_body(claim:, doc_type:, pdf_path:, file_number: nil, origina
     end
     # rubocop:enable Metrics/ParameterLists
 
-    def generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:)
+    def generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:, api_version:)
       # https://confluence.devops.va.gov/display/VAExternal/Document+Types
-      doc_type_to_form_name = {
-        'L075' => '21-22a',
-        'L122' => '526EZ',
-        'L190' => '21-22',
-        'L705' => '5103'
+      doc_type_names = {
+        'v1' => {
+          'L075' => 'representative',
+          'L190' => '21-22'
```

Comment:
Should `L190` also be `representative`?

The task doesn't explicitly say what `21-22` should be (only `21-22a`):

> We should update our mapping logic for v1 filenames to use "representative" instead of "21-22a"

But the AC does say:

> v1 PUT /2122 uses filename format "firstName_lastName_representative"

Maybe a Product question?

Reply:
that is a good question, opened a [slack thread](https://oddball.slack.com/archives/C019XKGS1CN/p1725044572459449) to discuss

Reply:
reworking this a little bit after getting clarification.  The number actually doesn't matter just the action it comes from.

Reply:
So this is only going to effect the `upload` action which is the POA `PUT`.  While ti does look like that has both options coming from the `PUT~, since we do not have the doc type it will always be `L075`.  That beng said I left the key in for both just to be safe.  But for this scenario that _shouldn't_ be possible.